blocks/arrow-icon/arrow-icon.css

```diff
@@ -0,0 +1,10 @@
+.arrow-icon {
+  font-size: 19px;
+  width: 34px;
+  height: 34px;
+  background: url("../../images/blue-arrow.png") center center no-repeat;
```

Comment:
It's better to place images into the block or element folder if that images used for styles